src/test/kotlin/net/maizegenetics/phgv2/pathing/MapKmersTest.kt

```diff
@@ -745,6 +749,20 @@ class MapKmersTest {
 
     }
 
+    @Ignore
+    @Test
+    fun testExtractKmersForPairedReads() {
```

Comment:
The chunks of code these 2 tests will eventually cover are covered by other tests so I am ignoring for now.  Will update these once the integration test is running smoothly.